packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.html

```diff
@@ -11,6 +11,8 @@
             data-ng-if="$ctrl.isFlavorCategoryIncluded(flavorByCategory.category)"
             data-id="{{flavorByCategory.category}}"
             data-heading="{{:: flavorByCategory.title }}"
+            data-subheading="{{:: flavorByCategory.isNew ? 'New' : '' }}"
```

Comment:
Shouldn't this be translated ?

Reply:
I thought the same, but New label in the notification is not translated.